Migrate transform pdf2parquet to Docling2parquet without notebooks. Also, delete the Windows instructions.

shahrokhDaijavad opened the pull request:
## Why are these changes needed?


## Related issue number (if any).

shahrokhDaijavad committed:
global change of pdf2parquet to docling2parquet

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
put back examples folder to its original form

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
removed the unnecessary comment line out of ray for html2parquet

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
DCO problem

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

touma-I commented:
@dolfim-ibm Please review when you get a chance. Thanks. 
Still looking for ways where older notebooks using the pdf2parquet get a message about it being deprecated and switch over to docling2parquet.

shahrokhDaijavad committed:
change docling2parquet to Docling2Parquet for class names

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad commented:
@dolfim-ibm can you please check if I did the change (docling2parquet to Docling2Parquet) correctly for "ALL" the classes?

dolfim-ibm commented on transforms/language/docling2parquet/README.md:
```diff
@@ -126,12 +126,12 @@ substituting the name of this transform image and runtime as appropriate.
+- [Integration test](test/test_docling2parquet.py)
 
-# Pdf2parquet Ray Transform 
+# docling2parquet Ray Transform 
```
```suggestion
# Docling2Parquet Ray Transform 
```

dolfim-ibm commented on transforms/language/docling2parquet/kfp_ray/README.md:
```diff
@@ -1,8 +1,8 @@
-# PDF2PARQUET Ray-base KubeFlow Pipeline Transformation 
+# docling2parquet Ray-base KubeFlow Pipeline Transformation 
```
```suggestion
# Docling2Parquet Ray-base KubeFlow Pipeline Transformation 
```

dolfim-ibm commented on transforms/transforms-1.0-lang.ipynb:
below there is an import to be updated:

```
from dpk_docling2parquet.transform_python import Docling2Parquet
```

dolfim-ibm commented on transforms/transforms-1.0-lang-ray.ipynb:
below there is an import to be updated:

```
from dpk_docling2parquet.transform_python import Docling2Parquet
```

dolfim-ibm commented on transforms/transforms-1.0-lang-ray-Windows.ipynb:
below there is an import to be updated:

```
from dpk_docling2parquet.transform_python import Docling2Parquet
```

dolfim-ibm commented on transforms/transforms-1.0-lang-Windows.ipynb:
below there is an import to be updated:

```
from dpk_docling2parquet.transform_python import Docling2Parquet
```

dolfim-ibm reviewed (commented):
Looks great.

I spotted only a few minor in the README, but also a few in the last notebooks.

shahrokhDaijavad committed:
Various docling case changes and changing pdf to document as needed

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
Resolve conflicting files and deleted Windows notebooks and reference

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
Resolve conflict

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
Changed test file names and chenged expected results

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
Changed all expected results

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad commented:
@dolfim-ibm I think I have made all the changes correctly, but I don't know why the test-src is failing. Can you please help by looking at the log of the test-src test? 
Another question: Because of the new column title of `document_convert_time`, there was a mismatch in the existing expected files, so I changed the notebook and created new output files that I moved to the `expected ` folder. I didn't know how to generate output for the 3 other `expectedxxx` folders, so I copied the same output to those folders too. 
In case it is easier to make changes yourself and try something instead of explaining what I should try, I gave you collaborator access to my fork.

dolfim-ibm committed:
update tests results

Signed-off-by: Michele Dolfi <dol@zurich.ibm.com>
